### PR TITLE
Introduction to the project

### PR DESCRIPTION
**Pull request by widmogrod**
Address issues: #6 #2

### CODE REVIEW

**Comment by widmogrod** [comment]
@krtek4 if you could verify my poor grammar I would be much obliged.

**Comment by krtek4** [comment]
Will do, but probably not until Friday. It's hard to find time those days.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
-# fantasy-land
-Specification for interoperability of common algebraic structures in PHP
+# FantasyLand [![Build Status](https://travis-ci.org/functional-php/fantasy-land.svg?branch=master)](https://travis-ci.org/functional-php/fantasy-land)
+## Introduction
+
+Specification for interoperability of common algebraic structures in PHP.
```
Comment (krtek4) [inline README.md]: The sentence from Fantasy Land seems better to me :

"This project specifies interoperability of common algebraic structures in PHP."

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+The inspiration for this package is taken from fantastic project [fantasyland](https://github.com/fantasyland/fantasy-land) `[A]`.
```
Comment (krtek4) [inline README.md]: The inspiration for this package is the fantastic  [fantasyland](https://github.com/fantasyland/fantasy-land) project.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,37 @@
...
+
+You can find following algebraic structures:
```
Comment (krtek4) [inline README.md]: The repository contains the following algebraic structures:

File: README.md [hunk 4]
```diff
@@ -1,2 +1,37 @@
...
+- [Foldable](src/FantasyLand/Foldable.php)
+- [Traversable](src/FantasyLand/Traversable.php)
+- [Chain](src/FantasyLand/Chain.php)
+- [Monad](src/FantasyLand/Monad.php)
+- [Pointed](src/FantasyLand/Pointed.php)
+
+Each algebraic structure must obey some laws. 
+To simplify verification of those laws, 
+this project provides encapsulated property tests, 
+that can be used to easily verify correctness of a newly implemented structures.
```
Comment (krtek4) [inline README.md]: Each of those algebraic structures must obey some laws. 
In order to simplify their verification, the project provides encapsulated property tests.
They can be used to easily verify the correctness of newly implemented structures.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,37 @@
...
+this project provides encapsulated property tests, 
+that can be used to easily verify correctness of a newly implemented structures. 
+
+- [Setoid Laws](src/FantasyLand/Helpful/SetoidLaws.php)
+- [Monoid Laws](src/FantasyLand/Helpful/MonoidLaws.php)
+- [Functor Laws](src/FantasyLand/Helpful/FunctorLaws.php)
+- [Applicative Laws](src/FantasyLand/Helpful/ApplicativeLaws.php)
+- [Monad Laws](src/FantasyLand/Helpful/MonadLaws.php)
+
+Examples how to use property tests can be found in the [directory of examples](src/FantasyLand/Helpful/Tests)
```
Comment (krtek4) [inline README.md]: You can find exemples on how to use those tests in the  [directory of examples](src/FantasyLand/Helpful/Tests).

**Comment by widmogrod** [comment]
@krtek4 thank you for important review.

**Pull request merged**